build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+source ./.env
+set -e +u
+set -o pipefail
+
+IMG="lidofinance/oracle:latest"
+export DOCKER_CONFIG=$HOME/.lidofinance
+
+PUSH=false
+# https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
```

Comment:
We won't use command line arguments for automation scripts. Just ENVs for simplicity and "a bit better" security. CMD parsing should be removed.